butterknife-sample/build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.application'
+
+android {
+  compileSdkVersion rootProject.ext.compileSdkVersion
+  buildToolsVersion rootProject.ext.buildToolsVersion
+
+  compileOptions {
+    sourceCompatibility rootProject.ext.sourceCompatibilityVersion
+    targetCompatibility rootProject.ext.targetCompatibilityVersion
+  }
+
+  defaultConfig {
+    applicationId "com.example.butterknife"
+    minSdkVersion rootProject.ext.minSdkVersion
+    targetSdkVersion rootProject.ext.targetSdkVersion
+    versionCode 1
+    versionName "1.0.0"
+  }
+
+  packagingOptions {
+    exclude 'LICENSE.txt'
+    exclude 'APK LICENSE.txt'
+  }
```

Comment:
Remove this block. license is default added and don't know what "APK LICENSE" is